swap/swap.go

```diff
@@ -251,7 +251,7 @@ func (s *Swap) Add(amount int64, peer *protocols.Peer) (err error) {
 
 	// Check if balance with peer is over the disconnect threshold
 	balance := swapPeer.getBalance()
-	if balance >= s.params.DisconnectThreshold {
+	if balance >= int64(s.params.DisconnectThreshold) {
```

Comment:
This is what I mean, now we have to do the conversion on every `Add`, which is unnecessary, especially because it's always the same value.